evap/staff/tests/test_views.py

```diff
@@ -808,8 +808,13 @@ def helper_semester_state_views(self, evaluation, old_state, new_state):
         The following tests make sure the evaluation state transitions are triggerable via the UI.
     """
     def test_semester_publish(self):
-        evaluation = mommy.make(Evaluation, course=mommy.make(Course, semester=self.semester), state='reviewed')
+        participant1 = mommy.make(UserProfile, email="foo@example.com")
+        participant2 = mommy.make(UserProfile, email="bar@example.com")
+        course = mommy.make(Course, semester=self.semester, state='reviewed', participants=[participant1, participant2], voters=[participant1, participant2])
+        evaluation = mommy.make(Evaluation, course=course, semester=self.semester, state='reviewed')
+
         self.helper_semester_state_views(evaluation, "reviewed", "published")
+        self.assertEqual(len(mail.outbox), 2)
 
```

Comment:
could you please also add a test counting the emails if one or two of the respective checkboxes are not selected?